packages/amplify-category-storage/src/commands/storage/override.ts

```diff
@@ -0,0 +1,83 @@
+/*
+    entry code for amplify override root
+*/
+
+import { generateOverrideSkeleton, $TSContext, FeatureFlags, stateManager, pathManager } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import * as fs from 'fs-extra';
+import inquirer from 'inquirer';
+import { DynamoDBInputState } from '../../provider-utils/awscloudformation/service-walkthroughs/dynamoDB-input-state';
+import { DDBStackTransform } from '../../provider-utils/awscloudformation/cdk-stack-builder/ddb-stack-transform';
+import * as path from 'path';
+import { categoryName } from '../../constants';
```

Comment:
Do we need to have this generic here? can we not use AmplifyCategories.STORAGE instead ?